CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.8.2
+
+<!-- generated comparing v5.8.1..master -->
+
+_May 30, 2022_
+
+A big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐛 bug fixes and 📚 documentation improvements.
+
+### `@mui/material@5.8.2`
+
+- &#8203;<!-- 04 -->[system] Add `getColorSchemeSelector` util (#32868) @siriwatknp
```

Comment:
Shouldn't this go to @mui/system?

Reply:
Updated.

Reply:
This means we don't have anything to release in Material UI this time 😮